### PR TITLE
bug fixes for test failures

### PR DESCRIPTION
**Pull request by hakuna-matatah**
Issue #, if available:
I had these local bug fixes for a while now to fix some of the failures we have been seeing in our tests in the past.

Description of changes:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/tasks/generators/clusterloader/load.yaml [hunk 1]
```diff
@@ -105,7 +105,7 @@ spec:
       cat $(workspaces.source.path)/perf-tests/clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml
       fi
   - name: run-loadtest
-    image: public.ecr.aws/kit/clusterloader2:0213bea
+    image: 197575167141.dkr.ecr.us-west-2.amazonaws.com/clusterloader2:latest
```
Comment (ashishranjan738) [inline tests/tasks/generators/clusterloader/load.yaml:108]: why is this changed ? is this a version upgrade ?

File: tests/tasks/setup/eks/awscli-cp-with-vpc.yaml [hunk 2]
```diff
@@ -51,6 +51,10 @@ spec:
       aws eks $ENDPOINT_FLAG --region $(params.region) wait cluster-active --name $(params.cluster-name) 
       aws eks $ENDPOINT_FLAG update-kubeconfig --name $(params.cluster-name) --region $(params.region)
       # enable PD on the cluster 
+      kubectl api-versions 
+      kubectl api-resources
+      # To do - remove this, temporarily adding because apparently apiserver is not recongnizing deamonset for sometime inconsistently, need to see if livez and readyz are active
```
Comment (ashishranjan738) [inline tests/tasks/setup/eks/awscli-cp-with-vpc.yaml:56]: shouldn't we print the resources after sleep then ? so that get we can get better info on whether the resources are ready or not ?

File: tests/tasks/generators/clusterloader/load.yaml (hunk 1 above)
Comment (hakuna-matatah) [reply tests/tasks/generators/clusterloader/load.yaml:108]: I have applied an internal patch to fix the failures we are seeing for EKS scale tests. I have a PR in upstream and in the process to make it to upstream.

Comment (hakuna-matatah) [reply tests/tasks/generators/clusterloader/load.yaml:108]: we have been using this patch for a while now.

File: tests/tasks/setup/eks/awscli-cp-with-vpc.yaml (hunk 2 above)
Comment (hakuna-matatah) [reply tests/tasks/setup/eks/awscli-cp-with-vpc.yaml:56]: After 5mins, it unlikely any of the resources are not ready, I'm mostly interested before sleep because ideally CFN-wait should wait until cluster is ready. Does it makes sense ?

File: tests/tasks/generators/clusterloader/load.yaml (hunk 1 above)
Comment (ashishranjan738) [reply tests/tasks/generators/clusterloader/load.yaml:108]: okay. you can link that PR here for tracking.

File: tests/tasks/setup/eks/awscli-cp-with-vpc.yaml (hunk 2 above)
Comment (ashishranjan738) [reply tests/tasks/setup/eks/awscli-cp-with-vpc.yaml:56]: got it

File: tests/tasks/generators/clusterloader/load.yaml (hunk 1 above)
Comment (hakuna-matatah) [reply tests/tasks/generators/clusterloader/load.yaml:108]: https://github.com/kubernetes/perf-tests/pull/2178

**Review by natherz97 — APPROVED** [review]
